cmd/flexible-informer-gen/generators/generic.go

```diff
@@ -101,7 +101,7 @@ func (g *genericGenerator) GenerateType(c *generator.Context, t *types.Type, w i
 
 	orderer := namer.Orderer{Namer: namer.NewPrivateNamer(0)}
 	for groupPackageName, groupVersions := range g.groupVersions {
-		group := group{
```

Comment:
nit - but now that we're working in a repo without gofumpt, do we still want this change?  I'm only asking because there's probably a ton more things that gofumpt would complain about in here, so this change suddenly seems out of place with the PR